lib/mappers/pa_mapper.py

```diff
@@ -0,0 +1,122 @@
+from dplaingestion.selector import exists, getprop
+from dplaingestion.mappers.dublin_core_mapper import DublinCoreMapper
+from dplaingestion.utilities import iterify
+from dplaingestion.selector import exists, setprop
+from urlparse import urlparse
+from akara import logger
+
+
+class PAMapper(DublinCoreMapper):
+
+    dcmi_types = {
```

Comment:
Was this supposed to be a List (with `[]`)?  Also, would ideally be `DCMI_TYPES` because it's being used as a constant.